flow/layers/physical_model_layer.h

```diff
@@ -13,19 +13,22 @@ class PhysicalLayerShape;
 
 class PhysicalModelLayer : public ContainerLayer {
  public:
-  PhysicalModelLayer();
+  PhysicalModelLayer() : isRect_(false){};
   ~PhysicalModelLayer() override;
 
   void set_path(const SkPath& path) {
     path_ = path;
-#if defined(OS_FUCHSIA)
     SkRRect rrect = SkRRect::MakeEmpty();
     if (path.isRRect(&rrect)) {
-      frameRRect_ = rrect;
       isRect_ = rrect.isRect();
     } else {
       isRect_ = false;
-      frameRRect_ = SkRRect::MakeRect(path.getBounds());
+    }
+#if defined(OS_FUCHSIA)
+    if (path.isRRect(&rrect)) {
+      frameRRect_ = rrect;
```

Comment:
Now it looks like isRect_ will not be set to true in the Fuchsia case.  I'd recommend moving this to physical_model_layer.cc and having one code path for all platforms that calls path.isRRect(&frameRRect_) and sets isRect_ accordingly